protocol/consistencychecks.go

```diff
@@ -100,6 +101,51 @@ func (cc *ConsistencyChecks) HandleResponse(requestType int, msg *Response,
 	return CheckPassed
 }
 
+// HandleDirectorySTRs is supposed to be used by CONIKS clients to
+// handle auditor responses, and by CONIKS auditors to handle directory
+// responses.
+func HandleDirectorySTRs(requestType int, msg *Response, signKey sign.PublicKey,
```

Comment:
Since it is being used nowhere and we are working on this in #170, I would like to remove it from this PR.